README.md

```diff
@@ -7,9 +9,8 @@ This repo contains the tools and step by step instructions so that your users ca
 
 1) [References](#references) 
 2) [Requirements](#requirements)
-3) [Installing the SDK](#installing-the-sDK)
-4) [Profile Retrieval](#profile-retrieval)
-5) [Handling Users](#handling-users)
+3) [Installing the SDK](#installing-the-sdk)
+4) [Configuration](#configuration)
 6) [Running the Example](#running-the-example)
```

Comment:
Aren't we missing the 5) ?

Reply:
Don't mean to butt in, but you can change them all to `1)`, and the numbers are auto-updated 🙂

Reply:
See https://github.com/getyoti/yoti-dotnet-sdk/blob/master/README.md (and [raw markdown](https://raw.githubusercontent.com/getyoti/yoti-dotnet-sdk/master/README.md))

Reply:
Yeah, I figured Markdown doesn't care about the numbers but changing them all to 1 seems a bit extreme

Reply:
But then you don't ever have to worry about updating numbers if you add/remove list items in the future 😃

Reply:
Actually, we had a similar discussion at https://github.com/getyoti/yoti-doc-scan-android/pull/12 and decided we should just remove the numbers :grinning: